docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -83,7 +83,28 @@ include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client
 
 == Clustered caches
 
-IMPORTANT: Content needed here!
+Distributed Caching can be enabled on a cache by adding Clustered storage as a tier to the cache.
```

Comment:
This is a more general introduction to clustering and should live at the top of the page instead of inside the clustered caches section.